### ISSUE TITLE
Publish Angular V.5 Compatible version to NPM

### ISSUE THREAD

**Issue by EugeValeiras** [angular/flex-layout#508]
Please Publish the latest version

**Comment by aamir1995** [comment]
`2.0.0-beta.10-4905443` is the latest and compatible version with Angular V5, We're using it in our project.

**Comment by jotatoledo** [comment]
https://github.com/angular/flex-layout/issues/489

**Comment by Necroskillz** [comment]
It may be compatible, but in package.json it still has 4.x dependencies:
```
  "peerDependencies": {
    "@angular/core": "~4.4.4",
    "@angular/common": "~4.4.4",
    "rxjs": "^5.5.0"
  }
```

I run `yarn check` as part of the build and also on the build server, so I can't use it with angular 5 without turning off all of these checks :/

**Comment by afriedma** [comment]
Any update on release?

**Comment by ThomasBurleson** [comment]
Merged with SHA 62457a597244a9.

> Great work @CaerusKaru !

**Comment by ThomasBurleson** [comment]
The Beta.12 NPM publish should go out today.

**Comment by ThomasBurleson** [comment]
Fixed with today's 2.0.0-beta.12 release. See SHA 62457a5972
